### DISCUSSION TITLE
Form to Flowise

### DISCUSSION THREAD

**Discussion by TacioRibeiro** [FlowiseAI/Flowise#675] [Q&A]
Hello,

Do you know how to send the answers from a form (for example a google form) to the input of a flowise chain ?

**Comment by chungyau97** [comment]
Hi @TacioRibeiro,

Currently `Google Form` loader is not yet implemented in [LangChain JS](https://js.langchain.com/docs/modules/data_connection/document_loaders/)

**Comment by FixingPixels** [comment]
I've done it using Gravity Forms and Gravity Flow (WordPress). Everything works great, just have a hard time getting response as it's not properly formatted JSON.
